### PR TITLE
deps(std): bump dependencies and prefer their lighter submodules

### PR DESCRIPTION
**Pull request by BastiDood**
This PR bumps the `std` dependencies from `0.160.0` to the latest `0.180.0` (as of writing). Breaking changes upstream (i.e., renamed exports, moved files, etc.) have been addressed accordingly.

While I was bumping the versions, I also refactored some exports such that we now prefer importing submodules instead of the more general `mod.ts` files (which downloads a bunch of unneeded files).

### CODE REVIEW

**Comment by bombillazo** [comment]
Hey @BastiDood, thanks for the contribution, We'll try to review this and see if we can close it.

**Comment by bombillazo** [comment]
Hey @BastiDood , any chance you might have to update to use the latest std (0.214.0)?

**Comment by BastiDood** [comment]
Sounds great! Will definitely work on that over the weekend. I'll go rebase the branch while I'm at it. 👍

**Comment by BastiDood** [comment]
Hello there @bombillazo! It turns out I had some time to spare. This PR should be ready for testing now.

**Comment by bombillazo** [comment]
Awesome, Ill take a look at it in the weekend

**Review by bombillazo — APPROVED** [review]
Looks good.

**Comment by bombillazo** [comment]
@bartlomieju wanna take a look?

File: deps.ts [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
+export { copy } from "https://deno.land/std@0.214.0/bytes/copy.ts";
+export { crypto } from "https://deno.land/std@0.214.0/crypto/crypto.ts";
+export { delay } from "https://deno.land/std@0.214.0/async/delay.ts";
+export { bold, yellow } from "https://deno.land/std@0.214.0/fmt/colors.ts";
 export {
   fromFileUrl,
   isAbsolute,
   join as joinPath,
-} from "https://deno.land/std@0.160.0/path/mod.ts";
+} from "https://deno.land/std@0.214.0/path/mod.ts";
```
Comment (iuioiua) [inline deps.ts:14]: This could be split up further.

**Review by iuioiua — APPROVED** [review]
LGTM

**Review by bartlomieju — APPROVED** [review]
I asked @iuioiua to take a look since he's been doing a lot of work on `deno_std`, because of their LGTM, it also LGTM to me. 

Thank you @BastiDood @bombillazo

**Pull request merged**